cmd/kubeadm/app/constants/constants.go

```diff
@@ -191,6 +191,8 @@ const (
 	CoreDNS = "CoreDNS"
 	// KubeDNS defines a variable used internally when referring to the kube-dns addon for a cluster
 	KubeDNS = "kube-dns"
+	// MinimumKubeProxyComponentConfigVersion
+	MinimumKubeProxyComponentConfigVersion = "v1.9.0-alpha.3"
```

Comment:
follow the same pattern here and create the `*version.Version` type directly with `MustParseSemantic` so you don't have to do that with in the consuming code

Reply:
done